scenes/game/game.tscn

```diff
@@ -584,10 +586,10 @@ texture_pressed = ExtResource("4_5y76h")
 texture_hover = ExtResource("4_5y76h")
 
 [node name="PlayerZones" type="VBoxContainer" parent="."]
-offset_left = 200.0
-offset_top = 380.0
-offset_right = 240.0
-offset_bottom = 420.0
+offset_left = 208.0
```

Comment:
These things were pretty carefully placed, so unless you have a good reason to move them, don't.
If you did have a good reason to shift everything around, there is still a lot of stuff to test that is corner cases.
Thinking like everyone with sealed + overdrive + treasure knight boost that adds a button here.

Reply:
ok, I'll revert these. it does look pretty janky with the current alignments though

Reply:
You are welcome to change this stuff, I just want to be sure the changes are intentional and you're aware of the various test matrices (like different characters with companions and extra buttons).
I'm all for making it look better.

We can also fix bugs that people find later, it isn't too big a deal.